.drone.starlark

```diff
@@ -67,6 +67,36 @@ config = {
 				'webUIcliTwoFactorTOTP': 'webUIcliTOTP'
 			},
 		},
+		'webUI-PHP7.3': {
+			'suites': {
+				'webUITwoFactorTOTP': 'webUITwoFactTOTP',
+			},
+			'browsers': [
+				'chrome',
+				'firefox'
+			],
+			'servers': [
+				'daily-master-qa',
+				'10.3.0'
```

Comment:
Note: we have to mention 10.3.0 here, because PHP 7.3 requires core 10.3 (we cannot run anything with PHP  7.3 and core 10.2.1)